docs/docs/project.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: docs
+title: Project API
+permalink: project
+---
+
+# Project API
+
+Github4s supports the [Project API](https://developer.github.com/v3/projects/). As a result,
+with Github4s, you can interact with:
+
+- [Projet](#project)
+  - [List projects](#list-project)
+  - [Columns](#columns)
+    - [List project columns](#list-projects-columns)
+
+The following examples assume the following imports and token:
+
+```scala mdoc:silent
+import github4s.Github
+import github4s.GithubIOSyntax._
+import cats.effect.IO
+import scala.concurrent.ExecutionContext.Implicits.global
+
+implicit val IOContextShift = IO.contextShift(global)
+val accessToken = sys.env.get("GITHUB4S_ACCESS_TOKEN")
+```
+
+They also make use of `cats.Id`, but any type container `F` implementing `ConcurrentEffect` will do.
+
+LiftIO syntax for `cats.Id` and `Future` are provided in `GithubIOSyntax`.
+
+## Project
+
+### List projects
+
+Note: The Projects API is currently available for developers to preview. During the preview period,
+the API may change without advance notice. Please see the blog post for full details. To access the
```

Comment:
```suggestion
the API may change without advance notice. To access the
```

Reply:
Also we might move this Note at the top and have it there only once, wdyt?

Reply:
👍 Yeah, good idea.